src/providers/twitch/api/Kraken.hpp

```diff
@@ -53,8 +53,8 @@ class Kraken final : boost::noncopyable
 private:
     NetworkRequest makeRequest(QString url, QUrlQuery urlQuery);
 
-    QString clientId;
-    QString oauthToken;
+    QString kclientId;
+    QString koauthToken;
```

Comment:
same as above but it's a different random prefix?!

Reply:
Happy to rename, what would you prefer?

Reply:
Maybe `apiClientId` or `krakenClientId`. Probably the former since we're in the `Kraken` class